js/main.js

```diff
@@ -0,0 +1,17 @@
+const nav = document.querySelector('.nav-mobile');
+const navBtn = document.querySelector('.burger-btn');
+const allNavItems = document.querySelectorAll('.nav-mobile__item');
+
+const openNav = () => {
+	nav.classList.toggle('nav-mobile--active');
+    navBtn.classList.toggle('bbtn-animation-open')
+	allNavItems.forEach((item) => {
+        item.addEventListener('click', () => {
```

Comment:
You are adding event listener here, but you are not removing it at all. I'm not sure how this works, but won't this cause issues after second attempt to click through the nav?

Reply:
Everything works, I've tested it a lot. This event works only when someone clicks on one of the menu items, and that closes the navigation. You can only reopen menu by clicking on the burger button.